### PR TITLE
Allow event creation

### PR DESCRIPTION
**Pull request by krichprollsch**

### CODE REVIEW

File: src/EventParticipation.php [hunk 1]
```diff
@@ -70,10 +70,11 @@ public static function translateStatus($status)
...
-        return [
-            'email' => $this->getUser()->getEmail(),
-            'responseStatus' => $this->getStatus(),
-        ];
+        $export = ['email' => $this->getUser()->getEmail()];
+        if (null !== $this->getStatus()) {
+            $export['responseStatus'] = $this->getStatus();
+        };
+
+        return $export;
```
Comment (Taluu) [inline src/EventParticipation.php:78]: imo we should return the "undecided" status instead of not returning any statuses

Comment (krichprollsch) [inline src/EventParticipation.php:78]: I would like a way to return nothing at all for a patch :/

File: src/AbstractEvent.php [hunk 2]
```diff
@@ -149,7 +149,10 @@ protected static function buildDate(array $data)
     public function export()
     {
         return [
+            'summary' => $this->getName(),
             'description' => $this->getDescription(),
+            'start' => ['dateTime' => $this->getStart()->format('c')],
+            'end' => ['dateTime' => $this->getEnd()->format('c')],
```
Comment (Taluu) [inline src/AbstractEvent.php:155]: shouldn't you also expose the timezones ?

Comment (krichprollsch) [inline src/AbstractEvent.php:155]: Seems ok according with google documentation

> end.dateTime  datetime    The time, as a combined date-time value (formatted according to RFC3339). A time zone offset is required unless a time zone is explicitly specified in timeZone.

https://developers.google.com/google-apps/calendar/v3/reference/events/insert#request-body

Comment (Taluu) [inline src/AbstractEvent.php:155]: ok, as `c` adds the offset

**Comment by krichprollsch** [comment]
ty!

**Pull request merged**